### ISSUE TITLE
Unable to sign up

### ISSUE THREAD

**Issue by jnak** [shader-park/shader-park-website#121]
When I try to sign up, the form fails silently. This is the output of the console:

```
 Form submission canceled because the form is not connected
vue.runtime.esm.js:1897 TypeError: Cannot read properties of undefined (reading 'auth')
    at signUp (SignUp.vue:55:7)
    at it (vue.runtime.esm.js:1863:26)
    at HTMLButtonElement.n (vue.runtime.esm.js:2188:14)
    at HTMLButtonElement.o._wrapper (vue.runtime.esm.js:6961:25)
```

I tried different usernames and passwords with different complexity - I don't think it's a form validation issue.

Best,
J

**Comment by PWhiddy** [comment]
Hi, thanks for letting us know! We’re hoping to have a fix out soon. In the meantime I recommend just saving code to your own computer.

**Comment by torinmb** [comment]
hey, thanks for the heads up! We just deployed a fix :) 5f03d84560aadb22c0b76f65950ef04dff0e21c9
